scalene/scalene_profiler.py

```diff
@@ -1219,12 +1217,10 @@ def disable_signals() -> None:
             signal.signal(ScaleneSignals.malloc_signal, signal.SIG_IGN)
             signal.signal(ScaleneSignals.free_signal, signal.SIG_IGN)
             signal.signal(ScaleneSignals.memcpy_signal, signal.SIG_IGN)
-            Scalene.__alloc_sigq.stop()
-            Scalene.__memcpy_sigq.stop()
-            Scalene.__cpu_sigq.stop()
+            Scalene.stop_signal_queues()
         except BaseException:
             # Retry just in case we get interrupted by one of our own signals.
-            Scalene.disable_signals()
+            Scalene.disable_signals()   # FIXME this could loop with an error
```

Comment:
If this could really loop with an error, shouldn't we handle it?

Reply:
Sure, it wasn't my purpose with the branch, but I can do it.